index.html

```diff
@@ -65,16 +65,18 @@
             </nav>
         </header>
         <section>
-            <div class="container">
-             <form action="">
-            <div class="form-control">
-                <label for="name-box"></label><input type="text" id="name-box" placeholder="Enter identification..." required onfocus="this.placeholder=''"
-                                                     onblur="this.placeholder='Enter identification...'"/>
-                <input type="button" id="submit-button" onclick="toggle();" value="Submit" style="float: right;">
-            </div>
-            </form>
-            </div>
         <div class="msger">
+            <header class="msger-header" id="msger-header">
+                <div class="container">
+                    <form action="">
```

Comment:
Should <form> be indented here? Maybe it should be level with the div below it? Ot the div below it should be indented?

Reply:
You're right, the form doesn't need to be indented, I'll fix that!